### PR TITLE
Add issue to zkp section

### PR DESCRIPTION
**Pull request by OR13**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1026.html" title="Last updated on Feb 3, 2023, 4:56 PM UTC (499a36e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1026/c344fa0...499a36e.html" title="Last updated on Feb 3, 2023, 4:56 PM UTC (499a36e)">Diff</a>

### CODE REVIEW

**Comment by Sakurann** [comment]
I thought we agreed to replace the section with a concrete shorter text that I think @brentzundel is working on.

**Comment by OR13** [comment]
@Sakurann I have yet to see a PR, and I think this issue can be merged before or after he updates the section.

I still feel strongly that it is misleading to claim that W3C Verifiable Credentials support ZKPs when nothing standards track exists, and there are no normative requirements associated with ZKPs.

**Comment by brentzundel** [comment]
I am working on a PR, but agree that this PR can be merged at any time independent of the one I'm working on.

> there are no normative requirements associated with ZKPs.
The v1.1 spec has normative requirements for using VCs with ZKPs, which is why we can't just delete the section (please note, I am not saying they are 'good' or 'testable' normative requirements). I am planning to address them as part of my PR.

File: index.html
Comment (brentzundel) [inline index.html:2913]:
```suggestion
We anticipate this section will undergo major revision for v2 of the data model,
and may remove it if there remain no active work items related to the topic of
"Zero-Knowledge Proofs".
```

Comment (OR13) [reply index.html:2913]: > may remove it if there remain no active work items related to the topic of "Zero-Knowledge Proofs".

can we make this a tiny bit stronger?

Comment (brentzundel) [reply index.html:2913]:
```suggestion
We anticipate this section will undergo major revision for v2 of the data model,
and plan to remove the normative requirements and move it to the
implementation guide if there are no active work items related to the topic of
"Zero-Knowledge Proofs" when we transition v2 to CR.
```

**Review by msporny — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-02-08-vcwg#section3-1) on 2023-02-08

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.1. Add issue to zkp section (pr vc-data-model#1026)

_See github pull request [vc-data-model#1026](https://github.com/w3c/vc-data-model/pull/1026)._




**Manu Sporny:** might remove zkp section.  
… also have new P.R that changes the zkp section.  
… need to decide how we are going to proces these.  
… thoughts?.  

**Brent Zundel:** #1030 incorporates and slightly changes the wording of #1026.  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, decision to merge this before PR #1030 was approved on the last call, merging.

**Pull request merged**
